### PR TITLE
add github publish command

### PR DESCRIPTION
**Pull request by sherifnada**
originally from #1892 . need to get this in master for testing on the branch to pick it up

### CODE REVIEW

**Review by jrhizor — APPROVED** [review]

File: .github/workflows/publish-command.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+name: integration-test
```
Comment (cgardens) [inline .github/workflows/publish-command.yml:1]: is this supposed to be named integration test? or is supposed to publish?

Comment (sherifnada) [reply .github/workflows/publish-command.yml:1]: was misnamed here, fixed in https://github.com/airbytehq/airbyte/commit/c5dee4cc2e2e26d6b9fb0f2d1c47c74543754c4d

File: .github/workflows/publish-command.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          java-version: '14'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.7'
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - run: |
+          docker login -u airbytebot -p ${DOCKER_PASSWORD}
+          SKIP_TESTS=true ./tools/integrations/manage.sh publish airbyte-integrations/${{ github.event.inputs.connector }}
```
Comment (cgardens) [inline .github/workflows/publish-command.yml:46]: do we have a guarantee that integration tests have passed before running publish and skipping tests?

**Review by michel-tricot — COMMENTED** [review]
Will it trigger for anyone who write these comments or is there a list of people who can?

**Pull request merged**